chromium_src/chrome/browser/ui/views/tabs/tab_strip_layout_helper.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_TABS_TAB_STRIP_LAYOUT_HELPER_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_TABS_TAB_STRIP_LAYOUT_HELPER_H_
+
+#define BRAVE_UI_VIEWS_TABS_TAB_STRIP_LAYOUT_HELPER_SET_USE_VERTICAL_TABS \
```

Comment:
With below, we can avoid corresponding patch file :) 
```
#define UpdateIdealBounds                                                      \
  UnUsed() { return 0; }                                                       \
  void set_use_vertical_tabs(bool vertical) { use_vertical_tabs_ = vertical; } \
                                                                               \
 private:                                                                      \
  bool use_vertical_tabs_ = true;                                              \
                                                                               \
 public:                                                                       \
  int UpdateIdealBounds

#include "src/chrome/browser/ui/views/tabs/tab_strip_layout_helper.h"

#undef UpdateIdealBounds
```

Reply:
Cool! Done.